src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/rc_command_factory.cc

```diff
@@ -74,125 +174,123 @@ CommandSharedPtr RCCommandFactory::CreateCommand(
     const app_mngr::commands::MessageSharedPtr& message,
     app_mngr::commands::Command::CommandSource source) {
   if (app_mngr::commands::Command::SOURCE_HMI == source) {
-    return CreateHMICommand(message, source);
+    mobile_apis::messageType::eType message_type =
```

Comment:
@BSolonenko if if (app_mngr::commands::Command::SOURCE_HMI == source)
we should return get_**hm**i_creator_factory(function_id, message_type).create(message);

Reply:
@LuxoftAKutsan  fixed in 75224d1

Reply:
@LitvinenkoIra   fixed in 75224d1